mantidproject/mantid
# Suppress clang-tidy for generated code

It turns out that if `CMAKE_CXX_CLANG_TIDY` is not set when the target is created, the tool won't be run on the compiled files.

**To test:**

A code review should be sufficient. However, setting up a build to use `clang-tidy` via `-DENABLE_CLANG_TIDY=on` would be better.

*There is no associated issue.*

*Does not need to be in the release notes* because it is only for users.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
